docs/internals/protocol/messages.rst

```diff
@@ -305,6 +305,11 @@ Format:
     };
 
     enum Cardinality {
+
+        // Zero cardinality is used in statements which don't return
+        // any result, such as CREATE DATABASE
+        ZERO = 0x6e,
```

Comment:
I think `ZERO` is ambiguous here, because it still implies you get a result, it's just an empty set, but of a certain type.  Whereas, `CREATE DATABASE` does not return *at all*, not even an empty set.  So, something like `NO_RESULT` would be clearer.